plugins/rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -364,9 +366,12 @@ export class PolicesServer {
         }
       }
 
-      for (const role of roles) {
-        await this.enforcer.addOrUpdateGroupingPolicy(role, 'rest', false);
-      }
+      const metadata: RoleMetadataDao = {
+        roleEntityRef: roleRaw.name,
+        source: 'rest',
+        description: roleRaw.metadata?.description ?? '',
+      };
+      await this.enforcer.addGroupingPolicies(roles, metadata);
```

Comment:
I am fine with leaving this change.

Using just the below in a PUT request will change the source from `legacy` to `rest`.
```
{
    "oldRole":
        {
            "memberReferences": ["group:default/group-ab"],
            "name": "role:default/group-ab"
        },
    "newRole":
        {
            "memberReferences": ["group:default/group-ab"],
            "name": "role:default/group-ab"
        }
}
```

I think it would be best to capture all of the information around legacy in the documentation. Just haven't figured out where the best place would be. Or even if it would be worth doing at this moment.